rotkehlchen/tests/fixtures/db.py

```diff
@@ -40,9 +40,13 @@ def fixture_ignored_assets() -> list[Asset] | None:
 
 
 @pytest.fixture(name='user_data_dir')
-def fixture_user_data_dir(data_dir, username) -> Path:
+def fixture_user_data_dir(data_dir: Path, username: str, worker_id: str) -> Path:
     """Create and return the user data directory. Clean up old directory if existing"""
-    user_data_dir = data_dir / USERSDIR_NAME / username
+    if worker_id != 'master':
```

Comment:
same here. Would need a comment to explain why this is here and what it does. Best in the docstring.